### ISSUE TITLE
Overloaded members not allowed with properties of the same name [Line]

### ISSUE THREAD

**Issue by bradPeg** [twinbasic/twinbasic#1667]
Converted vb6 project in beta 382. 

several forms and user controls have this error.

**Overloaded members not allowed with properties of the same name [Line].** 

The error points to the form/usercontrol name. At least one of these has no reference at all to Line in it.

**Comment by fafalone** [comment]
Is this a separate issue from the MSCal issue?

If so can you check if any other non-default controls (i.e. not the built in ones or comctllib/mscomctllib) have the 'OleObjectBlob' property by examining them in a text editor? Or if you're able to share a minimal reproduction or the project.

**Comment by bradPeg** [comment]
Yes it's a different issue. I looked at the ocx files in notepad++ and I couldn't find any reference to blog. I also added all the controls shared by 2 affected  forms to a third form and reconverted. The third form had no errors. 
It's also in the ActiveXExtender 
![image](https://github.com/twinbasic/twinbasic/assets/125519212/2d0780a0-3f9d-4b60-bd47-9afeb598e851)
I'll try putting the user control in a new project and trim it down till it works.

**Comment by fafalone** [comment]
It wouldn't be in the ocx file it would be in the .frm

For example when you put MSChart on a .frm,

```
   Begin MSChart20Lib.MSChart MSChart1 
      Height          =   2475
      Left            =   3135
      OleObjectBlob   =   "Form1.frx":0000
      TabIndex        =   3
      Top             =   2925
      Width           =   5910
   End
   ```

That binary data blob is the only particularly suspicious looking thing, so I was wondering if it might be a common theme.

**Comment by bradPeg** [comment]
Ok I tracked it down and it seems the forms and controls have lines names Line
Begin VB.Line Line 
            BorderWidth     =   2
            Index           =   6
            X1              =   780
            X2              =   900
            Y1              =   2640
            Y2              =   2280
         End
         Begin VB.Line Line 
            BorderWidth     =   2
            Index           =   2
            X1              =   1800
            X2              =   1680
            Y1              =   2640
            Y2              =   2280
         End
         Begin VB.Line Line 
            BorderWidth     =   2
            Index           =   0
            X1              =   780
            X2              =   1800
            Y1              =   2640
            Y2              =   2640
         End
and that causes the error

**Comment by fafalone** [comment]
Yup doesn't even have to be an array or import; 

-New StdExe

-Add Line to form

-Rename Line1 to Line

and error triggered. Thanks for running that down.

**Comment by bradPeg** [comment]
no worries I'll find out what I can about the others
